ui/src/features/dataset/gallery/delete-media-item/alert-dialog-content.component.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { AlertDialog, Flex, Text } from '@geti/ui';
+
+import { useEventListener } from '../../../../hooks/event-listener/event-listener.hook';
+
+type AlertDialogContentProps = {
+    itemId: string;
+    onPrimaryAction: () => void;
+};
+
+export const AlertDialogContent = ({ itemId, onPrimaryAction }: AlertDialogContentProps) => {
+    useEventListener('keydown', (event) => {
+        if (event.key === 'Enter') {
+            event.preventDefault();
+            onPrimaryAction();
+        }
+    });
+
+    return (
+        <AlertDialog
```

Comment:
Suggestion:
Can we delete multiple? If so, can we handle singular/plural here?
Example:
Singular:
`Are you sure you want to delete "${item.name}"?`

Plural:
`Are you sure you want to delete ${items.length}?`